migrations/Version202102051019351101_taoQtiItem.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\model\ClientLibConfigRegistry;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoQtiItem\model\QtiCreatorClientConfigRegistry;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202102051019351101_taoQtiItem extends AbstractMigration
```

Comment:
Just to double check.

Will the configs be OK after a fresh installation?

Reply:
yes